sympy/polys/subresultants_qq_zz.py

```diff
@@ -326,6 +329,20 @@ def sylvester(f, g, x, method = 1):
             k = k + 1
         return M
 
+def res(f, g, x):
+    """
+    The input polynomials f, g are in Z[x] or in Q[x].
+
+    The output is the resultant of f, g computed by evaluating
+    the determinant of the matrix sylvestr(f, g, x, 1).
+
+    """
+    if f == 0 or g == 0:
+        print("The resultant of ",f," and ", g," is not defined")
```

Comment:
Should raise an exception here (like `ValueError("The resultant of %s and %s is not defined" % (f, g))`).